css/logo-nav.css

```diff
@@ -893,11 +892,14 @@ a {
         margin-top: 20px;
     }
 }
-@media (max-width: 768px) {
+@media (max-width: 767px) {
```

Comment:
Why do you change to 767 ? We already have a media query at 768, changing to 767 make 2 transition both at 767 and 768 which looks weird.